### PR TITLE
Fixed example

### PR DESCRIPTION
**Pull request by jonasfj**

### CODE REVIEW

**Review by t0xicCode — APPROVED** [review]
:shipit:

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Running the Example
+
+1. Install dependencies `yarn install`
+2. Build source `yarn run compile`
```
Comment (t0xicCode) [inline examples/README.md:4]: `yarn install` runs `npm run compile` from what I saw...

Comment (jonasfj) [reply examples/README.md:4]: Hmm, it didn't on my end... I think it does on some other packages because we have postinstall hooks

**Review by imbstack — APPROVED** [review]
lgtm!

**Pull request merged**